src/app/modules/main/activity_center/item.nim

```diff
@@ -0,0 +1,84 @@
+import strformat, stint
+import ../../shared_models/message_item_qobject
+
+type Item* = ref object
+  id: string # ID is the id of the chat, for public chats it is the name e.g. status, for one-to-one is the hex encoded public key and for group chats is a random uuid appended with the hex encoded pk of the creator of the chat
+  chatId: string
+  name: string
+  author: string
+  notificationType: int
+  timestamp: int64
+  read: bool
+  dismissed: bool
+  accepted: bool
+  messageItem: MessageItem
+
+proc initItem*(
```

Comment:
@jrainville since this item is a `ref object` we need `result = Item()` otherwise there is a crash if you run the app with an account which already has notifications.